### ISSUE TITLE
How to stop player movement to make a selector panel?

### ISSUE THREAD

**Issue by TotallyNotSethP** [Spu7Nix/SPWN-language#317]
In this code: https://spu7nix.net/spwn/#/triggerlanguage/7selectorpanel
The player doesn't stop moving and the level just finishes almost immediately. How to stop this?

**Comment by rayzwastaken** [comment]
i dont think you can stop player movement, what you can try doing is going far into the level editor and then placing a block like this:
![image](https://github.com/Spu7Nix/SPWN-language/assets/142281382/89301adc-f5d7-4796-842c-2f6ec7d5af53)
also make sure that the selector panel is following the players x axis so it doesent go off screen.
